crates/primitives/src/specification.rs

```diff
@@ -1,9 +1,10 @@
+#![allow(non_camel_case_types)]
```

Comment:
`rust-analyzer` gives a warning for me on the struct without this attribute at the module level